projects/plugins/jetpack/extensions/blocks/blogging-prompt/blogging-prompt.php

```diff
@@ -21,10 +21,12 @@
  * registration if we need to.
  */
 function register_block() {
-	Blocks::jetpack_register_block(
-		BLOCK_NAME,
-		array( 'render_callback' => __NAMESPACE__ . '\load_assets' )
-	);
+	if ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || Jetpack::is_connection_ready() ) {
```

Comment:
Oh I'm sorry, `Jetpack` isn't in the namespace here!

```suggestion
	if ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || \Jetpack::is_connection_ready() ) {
```